executor/common_bsd.h

```diff
@@ -389,8 +389,10 @@ static long syz_extract_tcp_res(volatile long a0, volatile long a1, volatile lon
 
 static void sandbox_common()
 {
+#if !SYZ_THREADED
```

Comment:
This code runs as part of syz-executor and as part of C reproducers. They have same modes, but these modes are controlled differently. C code uses these macros, as we craft code specifically for this single reproducer and its particular mode. While executor is the same build always, so it can't use macros and uses flags instead. To make things even more interesting, flags are not defined in C reproducer (cannot be used lexically).
I assume `syz-execprog -threaded=0` is still broken, and this mode is also used in repro extraction process.
This should be something along the following amusing lines:
```
#if !SYZ_THREADED
#if SYZ_EXECUTOR
	if (!flag_threaded)
#endif
		if (setsid() == -1)
			fail("setsid failed");
#endif
```

Reply:
Thanks, updated.